intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationshipHeader.java

```diff
@@ -51,6 +52,8 @@ public class AtlasRelationshipHeader extends AtlasStruct implements Serializable
     private String                                  label               = null;
     private AtlasObjectId                           end1                = null;
     private AtlasObjectId                           end2                = null;
+    private Map<String, Object> customRelationshipInfo;
```

Comment:
nit: Can you check `customRelationshipInfo` requires any initialization ?

Reply:
not required